_includes/contributor/2-2-submission.md

```diff
@@ -0,0 +1,3 @@
+{:.bs-callout .bs-callout-info}
+**v2.2 Contribution Notice:**
+As of September 15, 2018, to keep 2.2 and 2.3 releases aligned, we only accept pull requests to 2.2-develop that contain changes already existing in 2.3-develop. We recommend submitting pull requests to 2.3-develop then [back porting](#porting) the changes to 2.2-develop.
```

Comment:
Maybe "As of September 15, 2018, in an effort to keep v2.2 and v2.3 releases aligned, we will only accept pull requests..."

Reply:
Updated, thanks!